main/guides/coreeval/local-testnet.md

```diff
@@ -0,0 +1,33 @@
+# Create a Local Testnet
+
+To create, start, and deploy an Agoric contract to a local Agoric Testnet, do the following:
+
+
+1. Install *Go* if needed.
+
+	```
+    brew install go
+    ```
+2. Navigate to `<agoric-sdk>/packages/cosmic-swingset` and run `make`.
+3. Prepare the chain and solo nodes.
+
+	```jsx
+	make scenario2-setup BASE_PORT=8000 NUM_SOLOS=0
+	```
+4. Start the chain.
+
+	```jsx
+	make scenario2-run-chain
```

Comment:
can we use `agoric start local-chain` instead? I'd rather not promote the `scenario2` stuff in the cosmic-swingset `Makefile` to a product feature.

Reply:
We can, but let's do this in a separate PR; I was able to troubleshoot the "This project isn't in a folder created by agoric init" error, but I've run into a different error, and this one doesn't have a helpful error message. I'll ping you or mfig to work through it.

Reply:
All the "make scenario2" commands have been converted to various "agoric XXX" commands.